weave

```diff
@@ -988,12 +1023,12 @@ case "$COMMAND" in
             dns_args "$@"
         fi
         collect_cidr_args "$@"
-        shift $CIDR_COUNT
+        shift $CIDR_ARG_COUNT
         CONTAINER=$(docker run $DNS_ARGS -d "$@")
         create_bridge
-        get_ipam_cidr $CONTAINER
-        with_container_netns_or_die $CONTAINER attach $CIDR_ARGS
-        when_dns_running with_container_fqdn $CONTAINER put_dns_fqdn $CIDR_ARGS
+        ipam_cidrs $CONTAINER $CIDR_ARGS
+        with_container_netns_or_die $CONTAINER attach $CIDR_ARGS $IPAM_CIDRS
```

Comment:
A lot of noise is added by having to write `$CIDR_ARGS $IPAM_CIDRS` in so many places. That would go away of `$CIDR_ARGS` included `$IPAM_CIDRS`. Then `$IPAM_CIDRS` would only appear in a couple of places, on its own.

Also, it's rather misleading that `ipam_cidrs` rewrites `CIDR_ARGS`, which was already set by `collect_cidr_args`. Especially since the type is different, i.e. it's not really args anymore, but just cidrs. So I suggest `ipam_cidrs` sets `ALL_CIDRS` and `IPAM_CIDRS`.